ocs_ci/ocs/acm/acm.py

```diff
@@ -0,0 +1,235 @@
+
+        """
+
+        self.import_cluster_ui(
+            cluster_name=cluster_name, kubeconfig_location=kubeconfig_location
+        )
+        for sample in TimeoutSampler(
+            timeout=450,
+            sleep=60,
+            func=validate_cluster_import,
+            cluster_name=cluster_name,
+        ):
+            if sample:
+                log.info(f"Cluster: {cluster_name} successfully imported")
+            else:
+                log.error(f"import of cluster: {cluster_name} failed")
+
+
+def copy_kubeconfig(file):
+    """
+
+    Args:
+        file: (str): kubeconfig file location
+
+    Returns:
+        list: with kubeconfig lines
+
+    """
+
+    try:
+        with open(file, "r") as f:
+            txt = f.readlines()
+            return txt
+
+    except FileNotFoundError as e:
+        log.error("file not found")
+        log.debug(f"expected file location {file}")
+        raise e
+
+
+def get_acm_url():
+    """
+    Gets ACM console url
+
+    Returns:
+        str: url of ACM console
+
+    """
+    mch_cmd = OCP(namespace=ACM_NAMESPACE)
+    url = mch_cmd.exec_oc_cmd(
+        "get route -ojsonpath='{.items[].spec.host}'", out_yaml_format=False
+    )
+    log.info(f"ACM console URL: {url}")
+
+    return f"https://{url}"
+
+
+def validate_page_title(driver, title):
+    """
+    Validates Page HTML Title
+    Args:
+        driver: driver (Selenium WebDriver)
+        title (str): required title
+
+    """
+    WebDriverWait(driver, 60).until(ec.title_is(title))
+    log.info(f"page title: {title}")
+
+
+def login_to_acm():
+    """
+    Login to ACM console and validate by its title
+
+    Returns:
+        driver (Selenium WebDriver)
+
+    """
+    url = get_acm_url()
+    log.info(f"URL: {url}, {type(url)}")
+    driver = login_ui(url)
+    validate_page_title(driver, title=ACM_PAGE_TITLE)
+
+    return driver
+
+
+def verify_running_acm():
+    """
+    Detect ACM and its version on Cluster
+
+    """
+    mch_cmd = OCP(namespace="open-cluster-management")
+    acm_status = mch_cmd.exec_oc_cmd(
+        "get mch -o jsonpath='{.items[].status.phase}'", out_yaml_format=False
+    )
+    assert acm_status == "Running", f"ACM status is {acm_status}"
+    acm_version = mch_cmd.exec_oc_cmd(
+        "get mch -o jsonpath='{.items[].status.currentVersion}'", out_yaml_format=False
+    )
+    log.info(f"ACM Version Detected: {acm_version}")
+
+
+def validate_cluster_import(cluster_name):
+    oc_obj = OCP()
+    log.debug({oc_obj.get(resource_name=ACM_MANAGED_CLUSTERS)})
+    conditions = oc_obj.exec_oc_cmd(
+        f"get managedclusters {cluster_name} -ojsonpath='{{.status.conditions}}'"
+    )
+    log.debug(conditions)
+
+    for dict_status in conditions:
+        log.info(f"Message: {dict_status.get('message')}")
+        log.info(f"Status: {dict_status.get('status')}")
+        assert dict_status.get(
+            "status"
+        ), f"Status is not True, but: {dict_status.get('status')}"
```

Comment:
Assert here checks if status exists and not for the status type. So the message should be something like:
```Failed to fetch the status of managed cluster {cluster_name}```